package.json

```diff
@@ -32,37 +32,34 @@
   },
   "homepage": "https://github.com/ChainSafe/discv5#readme",
   "devDependencies": {
-    "@types/assert": "^1.4.3",
     "@types/bn.js": "^4.11.5",
     "@types/chai": "^4.2.0",
+    "@types/debug": "^4.1.5",
     "@types/eslint": "^6.1.3",
     "@types/mocha": "^5.2.7",
     "@types/node": "^12.0.10",
-    "@types/randombytes": "^2.0.0",
-    "@types/secp256k1": "^3.5.0",
     "@typescript-eslint/eslint-plugin": "^2.7.0",
     "@typescript-eslint/parser": "^2.7.0",
     "chai": "^4.2.0",
     "eslint": "^6.6.0",
     "karma": "^4.3.0",
     "mocha": "^6.2.0",
     "nyc": "^14.1.1",
-    "ts-mockito": "^2.5.0",
     "ts-node": "^8.3.0",
-    "typescript": "^3.7.2"
+    "typescript": "^3.8.3"
   },
   "dependencies": {
-    "@types/debug": "^4.1.5",
-    "@types/ip": "^1.1.0",
-    "assert": "^2.0.0",
     "base64url": "^3.0.1",
     "bcrypto": "^4.2.8",
     "bigint-buffer": "^1.1.5",
     "debug": "^4.1.1",
     "dgram": "^1.0.1",
-    "ip": "^1.1.5",
+    "is-ip": "^3.1.0",
+    "libp2p-crypto": "^0.17.5",
+    "libp2p-crypto-secp256k1": "^0.4.3",
```

Comment:
I think libp2p-crypto already contains `libp2p-crypto-secp256k1`.
You can use it like `import {keys} from "libp2p-crypto"`

Reply:
Thanks. See https://github.com/libp2p/js-libp2p-crypto/pull/173 :joy_cat: